src/network.c

```diff
@@ -7,17 +7,18 @@
 
 #include "network.h"
 
+/* We need to make sure that we can support large bundles. */
+G_STATIC_ASSERT(sizeof(curl_off_t) == 8);
```

Comment:
@ejoerns Does this look good to you? I wanted to make extra sure that we don't break things with the new cases.

Reply:
Only a bit paranoid but it should not make things worse and in case this ever ever changes, we will get an early compile-time abort, this is ok.